pages/studentAssessmentInstanceHomework/studentAssessmentInstanceHomework.ejs

```diff
@@ -47,41 +47,54 @@
                 }) %>
             </div>
           </div>
-          <% if (assessment.group_work) { %>
-          <div class="container-fluid">
-            <div class="row">
-              <% if (groupinfo[0].student_authz_join) { %>
-              <div class="col-sm bg-light py-4 px-4 border">
-                <p>This is a group homework. Use your join code to invite others to join the group.</p>
-                <span class="badge badge-secondary">Group Name</span>
-                <%= groupinfo[0].name %><br>
-                <span class="badge badge-secondary">Join Code</span>
-                <%= joincode %><br>
-              </div>
-              <% } %>
-              <div class="col-sm bg-light py-4 px-4 border">
-                <% if (groupinfo[0].student_authz_leave) { %>
-                <form id="confirm-form" name="confirm-form" method="POST" class="d-flex justify-content-end">
-                  <input type="hidden" name="__action" value="leaveGroup">
-                  <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
-                  <button id="leave-group" type="submit" class="btn btn-danger">Leave the Group</button>
-                </form>
+        </div>
+        <% if (assessment.group_work) { %>
+          <div class="card-body border-top bg-light">
+            <div>
+              <p class="font-weight-bold mb-2">Group assessment</p>
+              <span>
+                <% if (groupinfo[0].student_authz_join) { %>
+                  You are in 
+                  <span class="badge badge-secondary">
+                    <%= groupinfo[0].name %>
+                  </span> 
+                  with
+                <% } else { %>
+                  Your teammates are
                 <% } %>
-                <span class="badge badge-secondary">Group Members</span>
-                <% groupinfo.forEach(function(row) {%>
-                <li><%= row.uid %></li>
-                <% }); %>
-              </div>
+              </span>
+              <%# Needs to be without whitespace (newlines) to avoid automatic spaces between list elements %>
+              <ul class="d-inline list-inline mb-0">
+                <% groupinfo.forEach(function(row) {%><li class="list-inline-item badge border"><%= row.uid %></li><% }); %>
```

Comment:
```suggestion
                <% groupinfo.forEach(function(row) {%><li class="list-inline-item"><a class="badge border" href="mailto:<%= row.uid %>"><%= row.uid %></a></li><% }); %>
```
What about making these badges clickable (`mailto`)? Just a thought, not particularly attached to the idea either way.

Reply:
Great idea!

Reply:
Are all uids email addresses though?

Reply:
Ah good point, not sure about that.

Reply:
Can always add it later if it's requested, but since we don't have a guarantee, we should probably keep it simple

Reply:
They aren’t necessarily email addresses. Illinois does use NetIDs that are identical to email addresses, but other schools don’t (e.g., UBC).  Also LTI users have non-email UIDs.